symbulate/random_variables.py

```diff
@@ -73,6 +73,29 @@ def sim(self, n):
 
         return RVResults(self.draw() for _ in range(n))
 
+    def __call__(self, input):
+        print(
+            "Warning: Calling an RV as a function simply applies the function defining\n"
+            + "the RV to the input, regardless of whether the input is a valid outcome in \n"
+            + "the probability space on which the RV is defined."
+        )
+        temp = self.probSpace.draw()
+        if isinstance(input, tuple):
+            if len(input) != len(tuple(temp)):
+                raise Exception("Input has wrong length")
+            if all(isinstance(x, type(temp[0])) for x in input):
+                return self.fun(input)
+            else:
+                raise Exception("Input of wrong type")
```

Comment:
This error message can be more specific, e.g., "Expected an input of type ..., got input of type ...."